<p>
  ProHits-viz was initially designed for the visualization of protein-protein
  interaction data, but it has always been able to visualize any tabular data that
  conforms to our <a href="https://prohits-viz.org/help/analysis/input-file">generic input format</a>.
  This format requires four parameters for each data point:
</p>
<ol>
  <li>the condition/experiment it belongs to</li>
  <li>the readout being quantified, usually a gene or protein name</li>
  <li>the value to be visualized, typically some sort of abundance metric</li>
  <li>a score indicating the significance of the value being visualized</li>
</ol>
<p>
  Many common proteomic and genomic experiments will generate data consistent with this format, or
  can be easily converted to it, and can thus benefit from the visualization tools we provide.
</p>
<p>
  To date, we have only provided explicit support for the output from two protein-protein
  interaction analysis tools: 1) SAINT and 2) CRAPome/REPRINT. Data output from other tools,
  whether from protein-protein interaction studies or some other experiment type,
  have required conversion by the user to conform with our input format.
</p>
<p>
  We are aiming to provide more direct support for the outputs from various analysis pipelines, allowing
  seamless integration with ProHits-viz, as we seek to make it as simple as possible for users to benefit
  from our tools. To this end, we have now added explicit support for
  the visualization of data output from common analysis tools that handle CRISPR experiments. These include:
</p>
<ul>
  <li><a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1015-8">BAGEL</a></li>
  <li><a href="https://genomemedicine.biomedcentral.com/articles/10.1186/s13073-019-0665-3">drugZ</a></li>
  <li><a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-014-0554-4">MAGeCK</a></li>
  <li><a href="https://journals.asm.org/doi/10.1128/MCB.00302-17">RANKS</a></li>
</ul>
<p>
  The metric to visualize from these tools typically takes both positive and negative values,
  for example a Z-score or log<sub>2</sub> fold change. Previously we required the
  &ldquo;abundance&rdquo; value to be visualized to only take non-negative values, but
  all of our tools have now been updated to support negative values as well. In some
  cases these values get treated as absolute/positive values, for example when calculating
  the dot size ratio on dot plots: a dot of -10 and +10 will be the same size; or when applying
  filters: a filter of 1 will now filter anything between -1 and 1, rather than just from 0 to 1.
  Otherwise visualization should be intuitive and as expected.
</p>
<p>
  The first step in using CRISPR data at ProHits-viz is to go to our
  <a href="https://prohits-viz.org/utilities">utilities page</a> and
  select the files you would like to analyze. You can select as many files as you would like.
  Then select the option
  &ldquo;Convert output from a CRISPR analysis tool to a ProHits-viz compatible format&rdquo;
  and the tool that produced the files. Hitting submit will return a single file
  containing all of the input analyses formatted for ProHits-viz. It is important to
  note that BAGEL, drugZ and RANKS analyze a single experiment at a time, and so if
  you wish to compare the results from multiple experiments, you must select all of the relevant
  files at the input stage.
</p>
<p>
  With the conversion complete, you can head to the
  <a href="https://prohits-viz.org/analysis">analysis page</a>, input the converted file
  and file type, and ProHits-viz will automatically select suitable options from here onwards
  for any type of analysis you would like to perform.
</p>